extensions/wikia/VisualEditorTourExperiment/VisualEditorTourExperiment.hooks.php

```diff
@@ -3,14 +3,20 @@
 
 class Hooks {
 
+	/**
+	 * Hook: BeforePageDisplay
+	 * Add VisualEditorTourExperiment assets and config var to pages where the editor can appear
+	 * @param \OutputPage $out
+	 * @param \Skin $skin
+	 * @return bool
+	 */
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)